### DISCUSSION TITLE
Nodes on the same machine

### DISCUSSION THREAD

**Discussion by Partymann2000** [pelican-dev/panel#547] [Ideas]
How about creating several nodes on a madrail so that you can divide your server into different nodes, e.g. for private and public servers?

**Comment by rmartinoscar** [comment]
Hey you should use Tags instead, Having multiple wings instances running at the same time on the same machine just for that reason seems like a hassle, you'll need 2 different ports, different directories and different config files

**Comment by QuintenQVD0** [comment]
diffrend docker networks ... and mutch more. Its max 1 node a machine and that wont change or ever be supported. IF you need that then make 2 vm's on that machine
